free-1.md

```diff
@@ -4,47 +4,143 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The project would serve people interacting with html and do web development. I
+think a language that can streamline the writing of html and its interaction
+with javascript and css would greatly benefit people that use these
+technologies. There currently are a few (many) javascript modules and libraries
+that try to do this. For example, jquery makes it easier to manipulate html on
+a webpage. However, as we saw in class, jquery's syntax was very verbose and
+at many points confusing. It has become bloated to become a large library. Most
+casual users who just want to build a simple website do not need jquery.
+
+I am looking more at something like JSX (https://facebook.github.io/jsx/).
+However, JSX only officially works within the React framework. I like JSX due to
+its clean syntax. If a DSL that uses JSX's syntax or something similar to
+interact directly with how to write HTML and javascript for a page, then it
+would be very nice.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
+experience like now? What would their experience be like if you could help
 them?_
 
+Writing HTML, CSS, and javascript is often more verbose than it needs to be,
+often very repetitive (all the closing tags and so forth). People often have to
+write long and winded HTML and CSS. Mix that with the fact that many normal
+javascript functions are extremely verbose. A DSL that can clean everything up,
+making all the components that are involved in web development easier to read
+and debug, woudl be very helpful.
+
+I am helping the people that do casual web development and don't necessarily
+need or want all the APIs the javascript language itself provides. Their
```

Comment:
I like the idea of making web development more accessible. I always felt it was pretty overwhelming with all the APIs and frameworks that are available. If you could market your language as a beginner- or casual developer-friendly alternative, it might make for a good entry point into the world of web dev.